nixos/modules/services/misc/gitlab.nix

```diff
@@ -475,6 +526,7 @@ in {
         gitAndTools.git
         openssh
         nodejs
+        procps
```

Comment:
This has always been necessary. If you go to Admin Area -> Monitoring -> Background Jobs, you get a 500 otherwise.